modules/scheduler/batch-job-template/templates/batch-job-base.json.tftpl

```diff
@@ -7,7 +7,7 @@
         },%{ endif }
         {
           "script": {
-            "text": "${runnable}"
+            "text": ${jsonencode(runnable)}
```

Comment:
e.g.
```json
            "text": "#!/bin/bash\n# docker run --gpus all \\\n#   -v /home:/home \\\n#   --entrypoint=\"/bin/bash\" \\\n#   nvcr.io/nvidia/cuquantum-appliance:22.07-cirq \\\n#   -c \"python3 /home/noise.py \u003e\u003e /home/${JOB_ID}.txt\"\nls /home \u003e /home/${JOB_ID}.txt\n"
```

Reply:
For sake of conversation, it will look weird in the sense of being escaped correctly.

```json
      "runnables": [
        {
          "script": {
            "text": "ls /home \u003e /home/${JOB_ID}.txt"
          }
        }
      ],
```

When submitted, you observed:

```yaml
    runnables:
    - script:
        text: ls /home > /home/${JOB_ID}.txt
```

Arguably this is weird, but less error-prone.

Reply:
The logs for the actual running job report

```
Executing script runnable script:{text:"ls /home > /home/${JOB_ID}.txt"} for Task task/j-e9cd0d08-39ab-45ba-a0c3-8abaf7552af3-group0-0/0/0.
```